src/js/utils/dom.js

```diff
@@ -791,10 +791,11 @@ export function isSingleLeftClick(event) {
     return true;
   }
 
-  // `mouseup` event on a single left click has
-  // `button` and `buttons` equal to 0
-  if (event.type === 'mouseup' && event.button === 0 &&
-      event.buttons === 0) {
+  // `mouseup` event on a single left click and
+  // `mousedown` event on pressing a button on the remote controller (Samsung TV)
+  //  have `button` and `buttons` equal to 0
+  if ((event.type === 'mouseup' || event.type === 'mousedown') &&
```

Comment:
Wonder if we should be adding a `click` type here too.
Also, whether this affects regular browsers and whether they will now preferentially treat `mousedown` as the click event rather than `mouseup`. I guess it depends on how `isSingleLeftClick` is used specifically. But it could have potential implications on accessibility.

Reply:
Yes, this is a 'dirty' hack that works but complicates the code and can introduce side effects on some exotic devices. I think it will be better to revert this code changes inside the 'isSingleLeftClick' function and do the change inside the seekbar component itself. Does it sound better for you?

Reply:
yeah, that does sound like a better approach. Keep hacks as contained as possible 🙂

Reply:
I have just updated the PR. Does it look better now?